### PR TITLE
Issue751 neishabouri counts documentation

### LINKED ISSUE #751: documentation on NeishabouriCounts
I believe we do not have any documentation on the use of NeishabouriCount_x, NeishabouriCount_y, and NeishabouriCount_z.

Also, i think there is a need for an explanation of how to use these in combination with published cut-points in cpm.

@jhmigueles Do you think this can be concisely explained in the documentation for argument "do.neishabouricounts"?

(EDIT) Also it would be good if this is discussed in the Cut-Points vignette, which is now the place we usually refer to when talking about cut-points.

### PR DESCRIPTION
**Pull request by jhmigueles**
fixes #751 -> improves documentation for the use of Neishabouri counts in GGIR.Rd and cut-points vignette.

Checklist before merging:

- [x] Existing tests still work (check by running the test suite, e.g. from RStudio).
- [ ] Added tests (if you added functionality) or fixed existing test (if you fixed a bug).
- [x] Updated or expanded the documentation.
- [x] Updated release notes in `inst/NEWS.Rd` with a user-readable summary. Please, include references to relevant issues or PR discussions.
- [ ] Added your name to the contributors lists in the `DESCRIPTION` and `CITATION.cff` files.

### CODE REVIEW

File: man/GGIR.Rd [hunk 1]
```diff
@@ -152,8 +152,11 @@ GGIR(mode = \Sexpr{format(formals(GGIR::GGIR)[["mode"]])},
 
       \item{acc.metric}{
         Boolean (default = \Sexpr{GGIR::load_params()[["params_general"]][["acc.metric"]]}).
```
Comment (vincentvanhees) [inline man/GGIR.Rd:154]: It is not a boolean but a character argument, correct?

**Review by vincentvanhees — CHANGES_REQUESTED** [review]
Looks good @jhmigueles, only one question.

Once the GitHub Actions issues is resolved, I will merge this.

File: man/GGIR.Rd (hunk 1 above)
Comment (jhmigueles) [reply man/GGIR.Rd:154]: Yes! You're right, I've fixed this now.

**Comment by vincentvanhees** [comment]
@jhmigueles I think I fixed the original problem in the workflow files. The reason we now still see failed tests is because it cannot install the R package cosinor on mac and Windows because that package is currently being updated.

https://cran.r-project.org/web/packages/cosinor/index.html
Once all releases of cosinor are live on all operating systems we should be able to run the tests again.

**Review by vincentvanhees — APPROVED** [review]

**Pull request merged**